analyzer/java/com/engflow/bazel/invocation/analyzer/bazelprofile/BazelProfile.java

```diff
@@ -185,12 +190,44 @@ public String getEmptyReason() {
 
   @Override
   public String getDescription() {
-    return "The Bazel profile.";
+    return "The JSON profile written by Bazel.";
```

Comment:
I think the original is better... this is a description of what this object provides... the JSON is what it consumes to create objects representing parts of a Bazel profile.

Reply:
I've removed the "JSON" part again.